tt_metal/llrt/rtoptions.cpp

```diff
@@ -120,6 +120,10 @@ RunTimeOptions::RunTimeOptions() {
     if (getenv("TT_METAL_SKIP_LOADING_FW")) {
         this->skip_loading_fw = true;
     }
+
+    if (getenv("TT_METAL_COMPILE_OUT_DEBUG_TOOLS_ON_DISPATCH_KERNELS")) {
```

Comment:
not a fan of super long env var names, but I didn't find anything much shorter than conveys what is going, or maybe: TT_METAL_NO_DEBUG_DISPATCH ?

Reply:
Thought quite about this as well and yeah ended up on what's there because it's the only thing I could think of that conveys the proper information. I feel like this env var is a niche case so having it be descriptive might be better than having it be short?

Reply:
* Infer dprint compile out from core specification
* Watcher disable dispatch to compile it out on dispatch cores TT_METAL_WATCHER_DISABLE_DISPATCH

Reply:
Should be updated in latest commit